docs/xml/collection-xmldata.xml

```diff
@@ -514,14 +514,23 @@
 						Each <code>&lt;lang/&gt;</code> node may have a <literal>percentage</literal> property, which describes the percentage value to which
 						a component has been translated.
 					</para>
+					<para>
+						The source locale for a component is assumed to be <code>en_US</code>. This is the language that the untranslated strings in the
+						component are written in. In GNU Gettext terminology, this is the <code>C</code> locale. If the component has a different source
+						language, it should be specified using the <code>source-locale</code> attribute on the <code>&lt;languages/&gt;</code> element.
+					</para>
+					<oara>
```

Comment:
Typo, makes the CI complain about this PR currently ;-)

Reply:
Oops